packages/flutter/test/material/slider_theme_test.dart

```diff
@@ -548,4 +548,244 @@ void main() {
     );
     await gesture.up();
   });
+
+  testWidgets('The slider track height can be overriden', (WidgetTester tester) async {
+    final SliderThemeData sliderTheme = ThemeData().sliderTheme.copyWith(trackHeight: 16);
+    double value = 0.25;
+    Widget buildApp({ bool enabled = true }) {
```

Comment:
Totally optional:

I know that you're following the existing pattern in these tests, but I feel like at least for the tests you added it would be better if this function was defined top-level in this file as `Widget _buildApp({ bool enabled = true }) { ...` so that it can be reused in your tests.

Reply:
I wouldn't mind replacing all of theses test's "buildApp" calls with one general one with 3 or 4 named params. Some start with enabled = true and some start with enabled = false. Doing so would make the initial call to pumpWidget per test more explicit too. Would that work?

Reply:
Done